### PR TITLE
Add `chainerx.ravel`

### PR DESCRIPTION
**Pull request by aksub99**
Add `chainerx.ravel`

### CODE REVIEW

File: chainerx_cc/chainerx/routines/manipulation.cc [hunk 1]
```diff
@@ -656,6 +656,11 @@ Array Swapaxes(const Array& a, int8_t axis1, int8_t axis2) {
     return out;
 }
 
+Array Ravel(const Array& a) {
+    Array out = a.Copy().Reshape({a.GetTotalSize()});
```
Comment (asi1024) [inline chainerx_cc/chainerx/routines/manipulation.cc]:
```suggestion
    Array out = a.Reshape({a.GetTotalSize()});
```

**Comment by aksub99** [comment]
I have made the change.

File: chainerx_cc/chainerx/routines/manipulation.cc [hunk 2]
```diff
@@ -656,6 +656,11 @@ Array Swapaxes(const Array& a, int8_t axis1, int8_t axis2) {
     return out;
 }
 
+Array Ravel(const Array& a) {
+    Array out = a.Reshape({a.GetTotalSize()});
```
Comment (asi1024) [inline chainerx_cc/chainerx/routines/manipulation.cc]: Why not write `return a.Reshape({...})` simply?

Comment (aksub99) [reply chainerx_cc/chainerx/routines/manipulation.cc]: Fixed.

**Comment by aksub99** [comment]
I am fixing the cpplint error now.

**Comment by asi1024** [comment]
@aksub99 Could you resolve conflict?

**Comment by asi1024** [comment]
Please remove `chainerx/manipulation/shape.py`.

**Comment by aksub99** [comment]
I have removed `shape.py` and resolved conflicts.

**Comment by asi1024** [comment]
pfnCI, test this please.

**Comment by asi1024** [comment]
Jenkins failure seems unrelated to this PR. LGTM.

**Pull request merged**